ionic2/app/pages/poke-detail/poke-detail.page.ts

```diff
@@ -1,18 +1,29 @@
-import {forwardRef} from "@angular/core";
```

Comment:
How exactly?
Isn't the poke-detail-page called only with the Id?

`this.navCtrl.push(PokeDetailPage, {pokemonId: pokemon.pokemonId});`